package.json

```diff
@@ -20,16 +20,20 @@
     "standard-http-error": "^2.0.1"
   },
   "devDependencies": {
-    "eslint": "^3.16.1",
-    "eslint-config-seegno": "^9.0.0",
-    "jest": "^19.0.2",
+    "eslint": "^6.5.1",
+    "eslint-config-uphold": "^1.0.1",
+    "jest": "^24.9.0",
     "koa": "^2.0.1",
     "lint-staged": "^3.4.0",
     "pre-commit": "^1.2.2",
+    "prettier": "^1.18.2",
     "supertest": "^3.0.0"
   },
   "engines": {
-    "node": ">=7.6"
+    "node": "^8.10.0 || ^10.13.0 || >=11.10.1"
+  },
+  "eslintConfig": {
```

Comment:
This seems to be against our conventions. Keep in `.eslintrc.yml`?

Reply:
Good point. It's best to have a single way to do things than to change depending on context.